### DISCUSSION TITLE
Assistance with Verification Process on GitHub Global Campus: Rejected ID and Documentation despite Multiple Attempt.

### DISCUSSION THREAD

**Discussion by DatUub** [community/community#54263] [GitHub Education]
### Hi everyone,

 I've been trying to verify my identity on the platform by submitting my ID, schedule, transcript, and interim report multiple times. However, all of my attempts have been rejected. All I get is this Generic Response:
Hey Ari Cartagena,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on May 2, 2023 for @AriCartagena.

Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.

**Comment by DatUub** [comment]
@samus-aran could you help me?

**Comment by samus-aran** [comment]
Hi @DatUub 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and I'll be happy to assist!
